src/content/github/common/consent.ts

```diff
@@ -0,0 +1,28 @@
+import { consentStorageKey, consentDialogCopy } from "./constants";
+
+export async function ensureConsent(
+  { checkOnly }: { checkOnly: boolean } = { checkOnly: false }
```

Comment:
checkOnly is a bit of a hack. We run two scripts on all github pages, one for files and one for PRs. If both run the popup you end up with two popups. So with this, one checks consent and shows the pop up, the other only checks.